liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/InstallFeatureMojo.java

```diff
@@ -59,7 +59,12 @@ private void installFeatures() throws PluginExecutionException {
             installFeaturesFromAnt(features.getFeatures());
         }
         else {
-            util.installFeatures(features.isAcceptLicense(), new ArrayList<String>(featuresToInstall));
+            if(util != null) {
```

Comment:
Just put `else (util != null)` at line 61 instead of a nested `if`. And no need for the following `else`. It will return automatically at the end of the method.